examples/plotting/time_distance.py

```diff
@@ -0,0 +1,95 @@
+"""
+=====================================================
+Creating a time-distance plot from a sequence of maps
+=====================================================
+
+This example showcases how you can use :func:`sunpy.map.pixelate_coord_path` and
+:func:`sunpy.map.sample_at_coords` on a sequence of images to create a time-distance diagram.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import propagate_with_solar_surface
+from sunpy.coordinates.screens import SphericalScreen
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+###############################################################################
+# First, we will need to acquire a sequence of images from SDO/AIA.
+
+query = Fido.search(
+    a.Time('2025-05-01 00:00:00', '2025-05-01 00:10:00'),
+    a.Instrument.aia,
+    a.Wavelength(171*u.angstrom),
+    a.Sample(1*u.min),
```

Comment:
I'd suggest using the same date as in the issue but possibly for shorter period of time at high cadence to keep the number of files down